logging/cluster-logging-support.adoc

```diff
@@ -30,6 +19,20 @@ The {logging-title} is not:
 * A guaranteed log sink
 * Secure storage - audit logs are not stored by default
 
+[id="cluster-logging-support-CRDs"]
+== Supported API custom resource definitions
+
+LokiStack development is ongoing. Not all APIs are supported currently supported.
+
+[options="header"]
+|=====================================================================
+| CustomResourceDefinition (CRD)| ApiVersion           | Support state
+| LokiStack      | lokistack.loki.grafana.com/v1       | Supported in 5.5
+| RulerConfig    | rulerconfig.loki.grafana/v1         | Supported in 5.7
+| AlertingRule   | alertingrule.loki.grafana/v1        | Supported in 5.7
+| RecordingRule  | recordingrule.loki.grafana/v1       | Supported in 5.7
+|=====================================================================
```

Comment:
Should reformat this with the other style table (example [here](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#alternative-footnote-styling-in-tables), ignore the footnotes bit)